stages/org.osbuild.coreos.live-iso

```diff
@@ -357,7 +376,8 @@ def main(inputs, devices, tree, options):
     finally:
         if umount_needed:
             subprocess.check_call(['umount', '-R', tmp_squashfs_dir])
-
+            subprocess.check_call(['umount', '-R',  f"{tmp_merged_dir}/boot"])
```

Comment:
We shouldn't need this one. It should be caught as part of the `-R` in the next one.